src/ssr/src/extractStyles.js

```diff
@@ -16,6 +16,7 @@ export default function extractStyles() {
   const scriptProps = {
     type: 'application/json',
     id: 'evergreen-hydrate',
+    nonce: options.nonce,
```

Comment:
Actually one last thing – can we only apply `nonce` when it is provided?

Reply:
@mshwery np, done